_docs/style/text-conventions/date-time.md

```diff
@@ -0,0 +1,215 @@
+- Avoid references to standard and daylight saving time because the appropriate designation changes frequently. However, if you need to include such a reference, insert *S* (for standard) or *D* (for daylight) as the second character in the abbreviation.
+- When referring to 12 a.m., use **12 midnight** or just **midnight**. When referring to 12 p.m., use **12 noon** or just **noon**.
+
+Consider the following examples for 12-hour clock time:
+
+- 10:29 a.m.
+- 6 p.m.
+- 6:00 p.m. to 8:30 p.m.
+- 10:30 a.m. central time (CT)
+- 1:30 p.m. central standard time (CST)
+- midnight
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+### 24-hour clock
+
+When you use the 24-hour clock to show time, use the following guidelines:
+
+- Use a colon to separate the hours, minutes, and seconds.
+- Show the hours, minutes, and seconds with two digits each, even if the leading digit is 0.
+- When possible, display the local time as specified in user preferences. If time preferences have not been set, use the assumed time based on the browser. Only use UTC when no timezone preference can be determined or standardization across timezones is necessary.
+- If you need to show a time zone, use Coordinated Universal Time (UTC), and indicate the time-zone offset from UTC.
+
+Consider the following examples for 24-hour clock time:
+
+- 08:29:37
+- 18:30:59
+- 18:00:00 to 20:30:00
+- 10:30:00 (UTC -6) (refers to CT)
+- 12:00:00 (noon)
+- 00:00:00 (midnight)
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Timestamp
+
+Add seconds or milliseconds to the time only when that level specificity is important.
+
+For example, use the following convention for timestamp:
+
+- Apr 2, 2016 12:59:16 AM CDT (UTC-5).
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Timeago
+
+Timeago is a measure of the length of time that has elapsed since an event has taken place (for
+example, a monitoring alarm or ticket update). Timeago helps users understand the timeline of events.
+
+Use the following conventions for elapsed time:
+
+- Under 1 minute: **just now**
+- After first minute: **1 min**
+- After second minute: **\<#\> mins**
+- After first hour: **1 hr**
+- After second hour: **\<#\> hrs**
+- After 24 hours: **1 day**
+- After 48 hours: **\<month (mixed case 3 letter abbreviation)\> \<numerical day\>** (for example, "Nov 3")
+- Hover tooltip with **\<timestamp\>**
```

Comment:
@palindromebart, this bulleted list came from the rackspace style guide. I'm unclear about the tooltip convention and if that will make sense to our audience.